raiden/network/transport/matrix.py

```diff
@@ -127,14 +127,22 @@ class MatrixTransport:
     def __init__(self, config: dict):
         self._bound_logger = None
         self._raiden_service: RaidenService = None
+
+        def http_retry_delay(prev_delay: float, count: int) -> float:
+            if count < 5:
```

Comment:
This is a nice candidate for a ternary operator which can turn this into an one liner

Reply:
Yes, but usually on these big `lambda`s, @hackaugusto comes to tell me to put it in a separate funcion =P

Reply:
Hahaha it depends ... if it's hard to read it's good to go on a separate function. But I think this is really simple and can be a ternary operator.

Also another comment. The hard-coded 5, please put it in constants.py or somewhere where you want to keep constants like this